main/main.go

```diff
@@ -44,6 +46,19 @@ func main() {
 	log.Info("Done executing go binary")
 }
 
+func configureLogLevel() {
+	logLevel := os.Getenv("CLOUDCONCIERGE_LOG_LEVEL")
+	if logLevel == "" {
+		logLevel = "info"
+	}
+	level, err := log.ParseLevel(logLevel)
+	if err != nil {
+		log.SetLevel(log.InfoLevel)
+	} else {
+		log.SetLevel(level)
+	}
```

Comment:
```suggestion
        logLevel := "info"
	envLogLevel os.Getenv("CLOUDCONCIERGE_LOG_LEVEL")
	if envLogLevel == "debug" {
		logLevel = "debug"
	}
	level, err := log.ParseLevel(logLevel)
	if err != nil {
		log.SetLevel(log.InfoLevel)
	} else {
		log.SetLevel(level)
	}
```
Slight change in logic to default to "info" and then change to"debug" only if explicitly received through the env var.